### PR TITLE
Require Load Balancer for all App Engine traffic

### PR DESCRIPTION
**Pull request by brunobowden**
- App Engine ingress set to `internal-and-cloud-load-balancing`
- Disables who-mh-hack.appspot.com domain
- Partial fix for #649

## How did you test the change?

```
server/bin/deploy-server.sh who-mh-hack
```

## Checklist:

- [x] Followed the [Contributor Guidelines](https://github.com/WorldHealthOrganization/app/blob/master/docs/CONTRIBUTING.md) and verified that all contributions are properly licensed pursuant to the [LICENSE](https://github.com/WorldHealthOrganization/app/blob/master/LICENSE).

### CODE REVIEW

**Review by matthewblain — APPROVED** [review]
I'm surprised that this does not seem possible in Terraform. Would you like (or like me) to file a bug? The app_engine_split_traffic resource is half way there. Using gcloud from terraform (as in PR 1752) feels like a hack. In fact, I'd think this entire script should be doable in Terraform, without any 'gcloud' commands, though I don't know if Terraform yet supports all the things required.

**Comment by matthewblain** [comment]
Also can you title this 'Require Load Balancer....'
(The LB was configured elsewhere.)

**Comment by brunobowden** [comment]
@matthewblain - please file a bug. I agree that it should all be under Terraform. Ideally you'd specify a tag or commit sha. Then as soon as it's merges, Terraform would run, do the pinned build, update the server binaries and the server state at the same time.

For now though, we need to focus on getting something out earlier by which we can do more good.

@yukuairoy - need your approval too

**Review by yukuairoy — APPROVED** [review]

**Comment by brunobowden** [comment]
@matthewblain - the idea is to put all the infrastructure under source control. Not just the configuration but also the deployment. So production and staging servers would all update automatically based on tag or sha which itself was under source control.

**Pull request merged**
